composer.json

```
@@ -6,19 +6,22 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.9",
-        "php": ">=7.2"
+        "php": ">=7.3",
+        "drupal/core-recommended": "^8.9 || ^9 || ^10"
```

```suggestion
        "drupal/core": "^8.9 || ^9"
```
we don't guarantee support of Drupal 10